test/config/benchmark_config.py

```diff
@@ -0,0 +1,11 @@
+from settings import * 
+from on_http import Configuration, ApiClient
```

Comment:
if you are targeting using 1.1 API python client. The latest lib name  changed from `on_http` to `on_http_api1_1`.
Also, any special reason why not leverage existing `api1_1_config.py`